packages/cmf/src/settings.js

```diff
@@ -21,6 +21,19 @@ export function generateDefaultViewId(viewId, componentName, componentId) {
 	return viewId;
 }
 
+/**
+ * this function remove all HOC from the componentName
+ * @param {String} viewId Connect(CMF(Container(MyComponent)))#default
+ * @return {String} MyComponent#default
+ */
+function withoutHOC(componentName) {
+	return componentName
```

Comment:
It seems that you don't like the regular expression :)

Reply:
No need for Regexp if they are not making more sense than simple string manipulation or are significantly faster.

Reply:
split is lower than regexp. https://jsperf.com/split-regexp42
IMO, to get something in a string, I prefer use a regexp.

Reply:
done